HelloWorldVue/src/components/Login.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+    <div class="login-container">
+        <h1>Login</h1>
+        <div class="form-container">
+            <form>
+                <div class="mb-3">
+                    <input
+                        type="email"
+                        class="form-control"
+                        id="exampleInputEmail1"
+                        aria-describedby="emailHelp"
+                        placeholder="Email or username"
+                    />
```

Comment:
We should try to write these in a single line.

Reply:
Thanks a lot for your valuable review @meet-aniket. I'll surely keep all this in mind next time.